### PR TITLE
Feature/markers origin destination

### PR DESCRIPTION
**Pull request by flibbertigibbet**
* Show markers on trip origin and destination points.
* Clear query results on tab switch
* Do not show query results if user switched to the other tab

### CODE REVIEW

File: src/app/scripts/cac/control/cac-control-sidebar-directions.js [hunk 1]
```diff
@@ -92,6 +93,14 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeOptions, MapTemplates
         bikeOptions.changeMode(options.selectors);
     }
 
+    function clearDirections() {
+        mapControl.setOriginDestinationMarkers(null, null);
+        mapControl.clearItineraries();
+        itineraryListControl.hide();
+        directionsListControl.hide();
+        $('.directions').removeClass('show-results');
```
Comment (kshepard) [inline src/app/scripts/cac/control/cac-control-sidebar-directions.js]: This should be defined in the selectors object. Same goes for the code not added in this PR that adds the class.

File: src/app/scripts/cac/map/cac-map-control.js [hunk 2]
```diff
@@ -282,4 +293,56 @@ CAC.Map.Control = (function ($, L, _) {
         map.panTo(latLng);
     }
 
+    /**
+     * Show markers for trip origin/destination.
+     * Will unset the markers if either co-ordinate set is null/empty.
+     *
+     * @param {Array} originCoords Start point co-ordinates [lat, lng]
+     * @param {Array} destinationCoords End point co-ordinates [lat, lng]
```
Comment (kshepard) [inline src/app/scripts/cac/map/cac-map-control.js]: Typo (I think): no hyphen in coordinates.

**Comment by kshepard** [comment]
+1

File: src/app/scripts/cac/map/cac-map-control.js (hunk 2 above)
Comment (flibbertigibbet) [inline src/app/scripts/cac/map/cac-map-control.js]: Different spelling of the same, but ok.

**Pull request merged**